<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>708</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Calibri</family>
    <pointsize>11</pointsize>
    <weight>9</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#C2E4E5;
font: 75 11pt &quot;Calibri&quot;;</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#15919B;
border-radius:20px;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QWidget" name="logo_img" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QFrame" name="frame_4">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">
color:white;
</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item alignment="Qt::AlignLeft">
             <widget class="QFrame" name="frame_7">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item alignment="Qt::AlignLeft">
                <widget class="QFrame" name="dashboard_logo">
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                </widget>
               </item>
               <item alignment="Qt::AlignLeft">
                <widget class="QPushButton" name="dashboard_btn">
                 <property name="styleSheet">
                  <string notr="true">font-style:&quot;Alconica&quot;;</string>
                 </property>
                 <property name="text">
                  <string>DASHBOARD</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item alignment="Qt::AlignLeft">
             <widget class="QFrame" name="frame_8">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item alignment="Qt::AlignLeft">
                <widget class="QFrame" name="price_logo">
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                </widget>
               </item>
               <item alignment="Qt::AlignLeft">
                <widget class="QPushButton" name="accounts_btn">
                 <property name="text">
                  <string>PRICES</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item alignment="Qt::AlignLeft">
             <widget class="QFrame" name="frame_9">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item alignment="Qt::AlignLeft">
                <widget class="QFrame" name="inventory_logo">
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                </widget>
               </item>
               <item alignment="Qt::AlignLeft">
                <widget class="QPushButton" name="prices_btn">
                 <property name="text">
                  <string>INVENTORY</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item alignment="Qt::AlignLeft">
             <widget class="QFrame" name="frame_11">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item alignment="Qt::AlignLeft">
                <widget class="QFrame" name="order_logo">
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                </widget>
               </item>
               <item alignment="Qt::AlignLeft">
                <widget class="QPushButton" name="inventory_btn">
                 <property name="text">
                  <string>ORDERS</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item alignment="Qt::AlignLeft">
             <widget class="QFrame" name="frame_10">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item alignment="Qt::AlignLeft">
                <widget class="QFrame" name="and_logo">
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                </widget>
               </item>
               <item alignment="Qt::AlignLeft">
                <widget class="QPushButton" name="orders_btn">
                 <property name="text">
                  <string>REPORTS AND LOGS</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_5">
              <property name="styleSheet">
               <string notr="true">color:white;
</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item alignment="Qt::AlignRight">
                <widget class="QFrame" name="frame_12">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>100</height>
                  </size>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_8">
                  <item alignment="Qt::AlignRight">
                   <widget class="QFrame" name="sales_logo">
                    <property name="frameShape">
                     <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                     <enum>QFrame::Raised</enum>
                    </property>
                   </widget>
                  </item>
                  <item alignment="Qt::AlignRight">
                   <widget class="QPushButton" name="pushButton">
                    <property name="text">
                     <string>SALES REPORT</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item alignment="Qt::AlignRight">
                <widget class="QFrame" name="frame_14">
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_9">
                  <item alignment="Qt::AlignRight">
                   <widget class="QFrame" name="logs_logo">
                    <property name="frameShape">
                     <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                     <enum>QFrame::Raised</enum>
                    </property>
                   </widget>
                  </item>
                  <item alignment="Qt::AlignRight">
                   <widget class="QPushButton" name="pushButton_3">
                    <property name="text">
                     <string>ACTIVITY LOGS</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item alignment="Qt::AlignRight">
                <widget class="QFrame" name="frame_16">
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_10">
                  <item alignment="Qt::AlignLeft">
                   <widget class="QFrame" name="setting_logo">
                    <property name="frameShape">
                     <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                     <enum>QFrame::Raised</enum>
                    </property>
                   </widget>
                  </item>
                  <item alignment="Qt::AlignRight">
                   <widget class="QPushButton" name="pushButton_2">
                    <property name="text">
                     <string>SYSTEM SETTINGS</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_6">
              <property name="styleSheet">
               <string notr="true">color:white;
</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item alignment="Qt::AlignLeft">
                <widget class="QFrame" name="frame_18">
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_11">
                  <item alignment="Qt::AlignLeft">
                   <widget class="QFrame" name="archive_logo">
                    <property name="frameShape">
                     <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                     <enum>QFrame::Raised</enum>
                    </property>
                   </widget>
                  </item>
                  <item alignment="Qt::AlignLeft">
                   <widget class="QPushButton" name="pushButton_6">
                    <property name="text">
                     <string>ARCHIVE</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item alignment="Qt::AlignLeft">
                <widget class="QFrame" name="frame_13">
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_12">
                  <item alignment="Qt::AlignLeft">
                   <widget class="QFrame" name="account_logo">
                    <property name="frameShape">
                     <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                     <enum>QFrame::Raised</enum>
                    </property>
                   </widget>
                  </item>
                  <item alignment="Qt::AlignLeft">
                   <widget class="QPushButton" name="pushButton_4">
                    <property name="text">
                     <string>ACCOUNTS</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item alignment="Qt::AlignLeft">
                <widget class="QFrame" name="frame_15">
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_13">
                  <item alignment="Qt::AlignLeft">
                   <widget class="QFrame" name="logout_logo">
                    <property name="frameShape">
                     <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                     <enum>QFrame::Raised</enum>
                    </property>
                   </widget>
                  </item>
                  <item alignment="Qt::AlignLeft">
                   <widget class="QPushButton" name="pushButton_5">
                    <property name="text">
                     <string>LOGOUT</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
